dashboard-api.js

```diff
@@ -1,10 +1,11 @@
 require('es6-promise/auto');
 var isArrowFunction = require('is-arrow-function');
-var getUrlParam = require('get-url-param');
 var Websandbox = require('websandbox/dist/frame');
+var url = require('js-polyfills/url');
```

Comment:
@cy6erskunk isn't it much bigger than get-url-param?

Reply:
A couple more Kb to be gzipped? Yes, you're right. 
(none of IE support `URL`, so some polyfill has to be added if using `URL`)